spec/student_spec.rb

```diff
@@ -0,0 +1,20 @@
+require './person'
+require './student'
+
+describe Student do
+  student = Student.new(18, 'John', parent_permission: true)
+
+  context 'should create instance of Student Class' do
+    it 'returns correct name' do
+      expect(student.name).to eq('John')
+    end
+
+    it 'returns correct age' do
+      expect(student.age).to eq(18)
+    end
+
+    it 'should return correct data' do
+      expect(student.play_hooky).to eq '¯(ツ)/¯'
+    end
```

Comment:
- [ ] it would be better to add tests for `add_classroom` and also for `can_use_services` in order to cover all the methods for `can_use_services` you should test positive case and negative case